content/blog/todo.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Todo"
+date: 2022-11-21
+draft: false
+---
+
+**TODO fix title**
+
+**TODO fix publish date**
+
+**TODO fix filename**
+
+I was recently at inception and I wanted to write down everything I learned.
+
+### I should take as much risk as possible
+
+Ian and Greg. Worst case scenario is not bad at all.
```

Comment:
Hmmm--Ii don't think this is the thing im supposed to be edting.  Let me try again.....